src/serviceDesks/incontact/inContactServiceDesk.ts

```diff
@@ -78,7 +79,7 @@ class InContactServiceDesk implements ServiceDesk {
       });
       await request.json();
     } catch (error) {
-      return Promise.reject();
+      throw new Error(error);
```

Comment:
And same thing here, you can remove the `try...catch` and just let the error propagate upwards. You don't need to wrap and re-throw it. Just let it be thrown as normal.